prereqs/git-info/nuget-client.props

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project>
   <PropertyGroup>
-    <GitCommitHash>d535886475b1eeda1813eb1947af23bd046abb25</GitCommitHash>
-    <OfficialBuildId>20250416.1</OfficialBuildId>
-    <OutputPackageVersion>6.15.0-preview.1.7</OutputPackageVersion>
+    <GitCommitHash>86c695c5bdd02ca0aa735eed42508b57f695baa7</GitCommitHash>
+    <OfficialBuildId>6.15.0.31</OfficialBuildId>
```

Comment:
Looks like this change is causing the build break. cc @premun @mmitche

Reply:
This is only failing in the soure-build job because that's the UB jobs don't read from that property anymore and use unified build versioning.